pkg/asset/cluster/tfvars.go

```diff
@@ -286,6 +286,13 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 			ServiceAccount: string(sess.Credentials.JSON),
 		}
 
+		// See https://github.com/coreos/coreos-assembler/blob/master/doc/openshift-gcp-nested-virt.md
+		// and https://cloud.google.com/compute/docs/instances/enable-nested-virtualization-vm-instances
+		licenses := []string{}
+		if _, nestedvirt := os.LookupEnv("OPENSHIFT_INSTALL_OS_IMAGE_GCP_ENABLE_NESTED_VIRT"); nestedvirt {
+			licenses = append(licenses, "https://www.googleapis.com/compute/v1/projects/vm-options/global/licenses/enable-vmx")
+		}
+
```

Comment:
using env to configure the list of licenses is not great!

Reply:
Any alternative suggestions?

Reply:
> https://github.com/openshift/installer/pull/3430#issuecomment-611131084

cannot answer or suggest without details on ^^

Reply:
would a dedicated field in the the customizable install-config.yaml be an acceptable approach?